### PR TITLE
Prepare MP LRA for the releases under the MPWG

### LINKED ISSUE #342: Prepare MP LRA for the release under the MPWG
We need to integrate a set of changes required for the release of MP LRA 1.0 under the MP specification process (WG requirements).

### PR DESCRIPTION
**Pull request by xstefank**
Signed-off-by: xstefank <xstefank122@gmail.com>

Resolves #342

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -282,32 +344,74 @@
                         <exclude>.checkstyle</exclude>
                         <exclude>.factorypath</exclude>
                         <exclude>.editorconfig</exclude>
+                        <exclude>.github/**</exclude>
+                        <exclude>**/*.html</exclude>
+                        <exclude>**/license-efsl.adoc</exclude>
+<!--                        <exclude>**/*.asciidoc</exclude>-->
```
Comment (ochaloup) [inline pom.xml:350]: is this comment needed here?

**Review by ochaloup — APPROVED** [review]
Seems fine to me.

File: pom.xml [hunk 2]
```diff
@@ -170,12 +198,47 @@
...
+                    <version>${bnd-maven-plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <groupId>biz.aQute.bnd</groupId>
+                    <artifactId>bnd-baseline-maven-plugin</artifactId>
+                    <version>${bnd-baseline-maven-plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>build-helper-maven-plugin</artifactId>
```
Comment (rdebusscher) [inline pom.xml:240]: I think it good to define the locale for timestamp;

See warning when run build.

----
     [WARNING] Using platform locale (en_BE actually) to format date/time, i.e. build is platform dependent!
----

**Review by rdebusscher — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (xstefank) [reply pom.xml:240]: nice catch, I missed this in other specs too.

Comment (xstefank) [reply pom.xml:240]: I will also try to fix other warnings after this is merged.

**Review by rdebusscher — APPROVED** [review]

**Pull request merged**